File: problemas/1000/1009.cs
using System;

namespace uri1009
{
    internal static class Program
    {
        private static void Main()
        {
            Console.ReadLine();
            double salario = double.Parse(Console.ReadLine());
            double vendas = double.Parse(Console.ReadLine());
            Console.WriteLine($"TOTAL = R$ {(salario + vendas * 0.15):F2}");
        }
    }
}

File: problemas/1000/1009.dart
import 'dart:io';

void main() {
  stdin.readLineSync();
  final salario = double.parse(stdin.readLineSync());
  final vendas = double.parse(stdin.readLineSync());
  stdout.writeln('TOTAL = R\$ ${(salario + vendas * 0.15).toStringAsFixed(2)}');
}